package.json

```diff
@@ -10,44 +10,30 @@
     "url": "git://github.com/bigcommerce/stencil-utils.git"
   },
   "scripts": {
-    "test": "npx grunt eslint && npx grunt karma",
-    "lint": "npx grunt lint",
-    "build": "npx grunt"
+    "build": "webpack --config webpack.conf.js",
+    "test": "jest",
+    "lint": "eslint '**/*.js' --ignore-pattern node_modules/",
```

Comment:
@jairo-bc doesn’t it ignore node_modules by default?

Reply:
You are write, removing it